tests/unit/test_wheels.py

```diff
@@ -34,9 +35,31 @@ def test_build_and_upload_wheel():
             },
         )
 
-        wheels.upload_to_dbfs()
+        wheels.upload_to_dbfs(force_dependencies=False)
+        installation.assert_file_dbfs_uploaded(re.compile("wheels/databricks_labs_blueprint-*"))
+    for wheel in wheels._local_wheel:
+        assert not os.path.exists(wheel)
+
+
+def test_build_and_upload_wheel_with_no_dependency():
+    installation = MockInstallation()
+    product_info = ProductInfo.from_class(MockInstallation)
+
+    wheels = WheelsV2(installation, product_info)
+    with wheels:
+        remote_on_wsfs = wheels.upload_to_wsfs(force_dependencies=True)
+        installation.assert_file_uploaded(re.compile("wheels/databricks_labs_blueprint-*"))
```

Comment:
also assert that SDK is uploaded:
https://github.com/databrickslabs/blueprint/blob/main/pyproject.toml#L17